src/polyswarmd/eth.py

```diff
@@ -9,6 +9,7 @@
 from eth.vm.forks.constantinople.transactions import (ConstantinopleTransaction)
 from flask import current_app as app, Blueprint, g, request
 from hexbytes import HexBytes
+from typing import Dict, Any, List, Tuple, Type
 from jsonschema.exceptions import ValidationError
```

Comment:
Unrelated to this PR, I think we should consider doing this for performance: https://www.peterbe.com/plog/jsonschema-validate-10x-faster-in-python . This might be a bit of a performance bottleneck in some of our services.